v2/pkg/templates/types/types.go

```diff
@@ -34,6 +34,8 @@ const (
 	SSLProtocol
 	// name:websocket
 	WebsocketProtocol
+	// name:rdap
+	RDAPProtocol
```

Comment:
Can you add the value for the protocol to the protocol mappings structure as well?

https://github.com/projectdiscovery/nuclei/blob/22fe6105c1b1d53eb921170541cea203047073cd/v2/pkg/templates/types/types.go#L44

Since otherwise it shows blank string for the template type.

```
[2021-12-13 15:16:43] [basic-rdap-example] [] [info]  [2022-08-13T04:00:00Z,RESERVED-Internet Assigned Numbers Authority]
```

Reply:
done